changelog/1316.breaking.rst

```diff
@@ -0,0 +1 @@
+``TestReport.longrepr`` is now always an instance of ``ReprExceptionInfo``. Previously it was a ``str`` when a test failed with ``pytest.fail(..., pytrace=False)``.
```

Comment:
```suggestion
``TestReport.longrepr`` is now always an instance of ``ReprExceptionInfo``. Previously it was an ``str`` when a test failed with ``pytest.fail(..., pytrace=False)``.
```

Reply:
``str`` starts with 's' which is a consonant. Why do you think that 'an' is a proper spelling?

Reply:
I read it as ESS-TEE-ARR, but maybe others don't. Not a big deal either way :)